nomad/vault.go

```diff
@@ -1228,8 +1239,18 @@ func (v *vaultClient) EmitStats(period time.Duration, stopCh chan struct{}) {
 		case <-time.After(period):
 			stats := v.Stats()
 			metrics.SetGauge([]string{"nomad", "vault", "distributed_tokens_revoking"}, float32(stats.TrackedForRevoke))
+			metrics.SetGauge([]string{"nomad", "vault", "token_ttl"}, float32(tokenTTL(v.tokenData)/time.Millisecond))
```

Comment:
The metric here better has a server label to identify problematic nomad server - but looks like none of the other metrics have server id.  I went with consistency in this case, so we can address all metrics together later.  I suspect it'd be better to add a global label at the metrics agent level than pollute the metric key.

Users currently can set additional tags in the metrics agent level (e.g. datadog agent), or by setting server specific keys in the Nomad config (e.g. `statsd_tags`).